tests/unit/s2n_ecc_preferences_test.c

```diff
@@ -45,5 +45,18 @@ int main(int argc, char **argv)
         EXPECT_FALSE(s2n_ecc_preferences_includes_curve(&s2n_ecc_preferences_20201021, TLS_EC_CURVE_ECDH_X25519));
     };
 
+    /* Test: check all security policies don't contain the placeholder curve "s2n_ecc_curve_none" */
+    {
+        for (size_t policy_index = 0; security_policy_selection[policy_index].version != NULL; policy_index++) {
+            const struct s2n_security_policy *security_policy = security_policy_selection[policy_index].security_policy;
+            const struct s2n_ecc_preferences *ecc_preferences = security_policy->ecc_preferences;
+            EXPECT_NOT_NULL(ecc_preferences);
+
+            for (size_t curve_index = 0; curve_index < ecc_preferences->count; curve_index++) {
+                EXPECT_NOT_EQUAL(ecc_preferences->ecc_curves[curve_index], &s2n_ecc_curve_none);
+            }
+        }
+    };
```

Comment:
Nit: we'll likely reuse this loop for any other ecc preference validation we need to do. I'd suggest naming the test something more generic, like "validate all ecc preferences" and move this more specific test name down to line 56 where you make the assertion.